modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -27,7 +27,7 @@ In the table, features are marked with the following statuses:
 
 s|Version s|kam  s|Argo CD CLI  s|Helm  s|Kustomize s|Argo CD s|Argo Rollouts s|Dex     s|RH SSO |
 
-|1.15.0 |NA  |2.13.1 TP |3.15.4 GA |5.4.3 GA |2.13.1 GA |1.7.2 GA |2.41.1 GA |7.6.0 GA |4.14-4.17
+|1.15.0 |NA  |2.13.1 TP |3.15.4 GA |5.5.0 GA |2.13.1 GA |1.7.2 GA |2.41.1 GA |7.6.0 GA |4.14-4.17
```

Comment:
We should revert the kustomize version to `5.4.3 GA` as we found breaking changes with `5.5.0 GA`

Reply:
Fixed.